pkg/render/node.go

```diff
@@ -215,7 +216,8 @@ func (c *nodeComponent) Objects() ([]client.Object, []client.Object) {
 		objs = append(objs, c.nodePodSecurityPolicy())
 	}
 
-	objs = append(objs, c.nodeDaemonset(cniConfig))
```

Comment:
Let's just revert this to the original line so that it's consistent with the rest of the object rendering. 

```go
objs = append(objs, c.nodeDaemonset(cniConfig))
```